content/en/community/hackathons/usoc23/overview/index.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Session 01 - Overview of Unikraft
+linkTitle: Setting Up Unikraft
+weight: 5
+summary: |
+  We present the setup. The students will use the scripts to run and test all the applications available.
+  Expected time: 60 minutes
+---
+
+## Setup
+
+We encourage you to split into teams of 5-6 people, we have the teams proposals [here](https://docs.google.com/spreadsheets/d/1O17J63nHGo9ZVPQvDKCJbk2jUUB44D-rtyB1CmJlXFg/edit?usp=sharing), you can use the `#hack-team-XX-voice` channels on the [Discord server](https://bit.ly/UnikraftDiscord).
+You will need a Linux environment for this session.
+You can use a virtual machine, but we recommend a native Linux install.
+
+You will need to install the following packages:
+
+* `build-essential` / `base-devel` / `@development-tools` (the meta-package that includes `make`, `gcc` and other development-related packages)
+* `gcc-aarch64-linux-gnu`
+* `sudo`
+* `flex`
+* `bison`
+* `git`
+* `wget`
+* `uuid-runtime`
+* `qemu-system-x86`
+* `qemu-system-arm`
+* `qemu-kvm`
+* `sgabios`
+
+On Ubuntu, Debian, and other `apt`-based distributions, you can use the following command to install the requirements:
+
+```console
+$ apt-get install -y --no-install-recommends \
+        build-essential \
+        sudo \
+        gcc-aarch64-linux-gnu \
+        libncurses-dev \
+        libyaml-dev \
+        flex \
+        bison \
+        git \
+        wget \
+        uuid-runtime \
+        qemu-kvm \
+        qemu-system-x86 \
+        qemu-system-arm \
+        sgabios
+```
+
+For running Unikraft with networking support, you will also need to configure QEMU to allow network bridge access:
+
+```console
+$ sudo mkdir /etc/qemu/
+$ echo "allow all" | sudo tee /etc/qemu/bridge.conf
+```
+
+## Running `app-helloworld`
+
+The Unikraft `helloworld` application is located in [this repository](https://github.com/unikraft/app-helloworld), along with instructions on how to configure, build and run the application.
+We will go through them without focusing that much on the details, since we will see how everything works besides the scenes in the next sessions.
```

Comment:
```suggestion
We will go through them without focusing that much on the details, since we will see how everything works behind the scenes in the next sessions.
```